kustomize/v1alpha8/default/cluster-template.yaml

```diff
@@ -28,7 +28,8 @@ spec:
     kind: Secret
   apiServerLoadBalancer:
     enabled: true
-  managedSecurityGroups: true
+  managedSecurityGroups:
+    enabled: true
```

Comment:
Please could we add the Calico rules here? We'll likely tweak this at some point to compose them in with kustomize, but for now I think it's simplest just to include them in the default templates.